frontend/src/app/shared/pipes/cater-capacity.pipe.ts

```diff
@@ -0,0 +1,30 @@
+import type { PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
+import type { CanCaterFor } from 'digital-fuesim-manv-shared';
+
+export type CatererType = 'material' | 'personnel';
+
+@Pipe({
+    name: 'caterCapacity',
+})
+export class CaterCapacityPipe implements PipeTransform {
+    transform(canCaterFor: CanCaterFor, type: CatererType): string {
+        let result = '';
+        if (type === 'personnel') {
+            result += 'Diese Einsatzkraft ';
+        } else {
+            result += 'Dieses Material ';
+        }
+        if (canCaterFor.logicalOperator === 'or') {
+            result +=
+                'kann nur Patienten der selben Verletzungsschwere helfen.\n';
+        } else {
+            result +=
+                'kann Patienten verschiedener Verletzungsschweren helfen und dabei statt schwerer auch leichter verletzten helfen.\n';
+        }
+        result += `Rot:  ${canCaterFor.red}\n`;
+        result += `Gelb: ${canCaterFor.yellow}\n`;
+        result += `Grün: ${canCaterFor.green}`;
+        return result;
```

Comment:
Boulder

Instead of using a Pipe and formatting the text with `\n` I would use a Component instead. This will probably be more future-proof as the `canCaterFor` will probably get even more complicated in the future.

You could then reuse the `PatientStatusBadgeComponent` instead of `Rot`, `Gelb`, and `Grün`.
If you search the whole codebase for the German color names, you can see that there are no results. Therefore it would be inconsistent with the current UI to use them here to represent the triage categories.